docs/source/extension/extension_tutorial.rst

```diff
@@ -201,7 +201,7 @@ JupyterLab:
    On Windows, symbolic links can be activated on Windows 10 or above for Python version 3.8 or higher
    by activating the 'Developer Mode'. That may not be allowed by your administrators.
    See `Activate Developer Mode on Windows <https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development>`__
-   for instructions.
+   for instructions. Making this and runing ``jupyter labextension develop --overwrite`` is necessary for Windows 11, so that JupyterLab will use an updated build.
```

Comment:
I recently noticed that this information is not exposed properly too - thank you for proposing to emphasise it more!

> and runing ``jupyter labextension develop --overwrite`` is necessary for Windows 11, so that JupyterLab will use an updated build

This is in general true regardless of whether Windows 11 is used or any other OS. I would suggest not to include this sentence.

Instead, I would suggest that we make three different changes to address the underlying issue:
- let's mention the need to enable symbolic links in [Developing a prebuilt extension](https://jupyterlab.readthedocs.io/en/stable/extension/extension_dev.html#developing-a-prebuilt-extension) section; this could be a single sentence and a link to this note (you will need to create an anchor).
- lets reword this box to say that "links _need_ to be activated" rather than "links _can_ be activated"
- let's consider if we should change it from `note::` to `important::` admonition. Maybe it should even have a custom title, like "Important for Windows users".

Reply:
@krassowski sure! Thank you for suggestion.

Reply:
Dear @krassowski I have updated this PR according to your suggestions. Please, take a look. Thank you:)